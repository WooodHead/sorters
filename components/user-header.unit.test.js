import UserHeader from './user-header'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('user-header', () => {
    it('shallow-renders', () => {
        const component = shallow(<UserHeader
            name="Nick Redmark"
            username="nmaro"
            emailHash="foo"
            about="Hi there I'm Nick"
            route="journal"
        />)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()
    })
})
