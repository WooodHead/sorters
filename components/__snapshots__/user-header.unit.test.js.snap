// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user-header shallow-renders 1`] = `
<div>
  <h1>
    Nick Redmark
  </h1>
  <div
    className="row"
  >
    <div
      className="col-xs-6 col-sm-3 col-md-2"
    >
      <Gravatar
        default="retro"
        md5="foo"
        protocol="//"
        rating="g"
        size={200}
        style={
          Object {
            "display": "block",
            "height": "auto",
            "marginBottom": "1.5rem",
            "marginLeft": "auto",
            "marginRight": "auto",
            "width": "100%",
          }
        }
      />
    </div>
    <div
      className="col-xs-12 col-sm-9 col-md-10"
    >
      <p>
        <a
          href="/u/nmaro"
        >
          /u/
          nmaro
        </a>
      </p>
      <div>
        <Markdown
          content="Hi there I'm Nick"
        />
      </div>
    </div>
  </div>
  <ul
    className="nav nav-tabs"
  >
    <li
      className={false}
      role="presentation"
    >
      <a
        href="/u/nmaro"
      >
        ⛏ Activity
      </a>
    </li>
    <li
      className={false}
      role="presentation"
    >
      <a
        href="/u/nmaro/profile"
      >
        👤 Profile
      </a>
    </li>
    <li
      className={false}
      role="presentation"
    >
      <a
        href="/u/nmaro/goals"
      >
        ◎ Goals
      </a>
    </li>
    <li
      className="active"
      role="presentation"
    >
      <a
        href="/u/nmaro/journal"
      >
        ✎ Journal
      </a>
    </li>
    <li
      className={false}
      role="presentation"
    >
      <a
        href="/u/nmaro/reads"
      >
        📖 Books
      </a>
    </li>
    <li
      className={false}
      role="presentation"
    >
      <a
        href="/u/nmaro/topics"
      >
        💡 Topics
      </a>
    </li>
    <li
      className={false}
      role="presentation"
    >
      <a
        href="/u/nmaro/essays"
      >
        ✎ Essays
      </a>
    </li>
    <li
      className={false}
      role="presentation"
    >
      <a
        href="/u/nmaro/speeches"
      >
        👄 Speeches
      </a>
    </li>
    <li
      className={false}
      role="presentation"
    >
      <a
        href="/u/nmaro/conversations"
      >
        🗩 Conversations
      </a>
    </li>
  </ul>
</div>
`;
